server/modules/redis-client.spec.js

```diff
@@ -118,6 +128,19 @@ describe('The redis-client module', function() {
     this.sandbox.stub(mocks.wrappedRedisClient, 'del').callsFake((key, cb) => {
       cb();
     });
+    this.sandbox.stub(mocks.wrappedRedisClient, 'hgetall').callsFake((key, cb) => {
+      cb(null, { dopplerApiKey: 'jv8jf9a8jecdsc' });
+    });
+    this.sandbox.stub(mocks.wrappedRedisClient, 'smembers').callsFake((key, cb) => {
+      cb(null, [
+        'my-store.myshopify.com',
+        'my-store-2.myshopify.com'
+      ]);
+       
+    });
+    this.sandbox.stub(mocks.wrappedRedisClient, 'sadd').callsFake((key,obj, cb) => {
```

Comment:
callsFake((key,obj, cb) 
                      | missing space between _,_ and **obj**